### PR TITLE
MES-2577 - Fixing TODO's in Service

### PR DESCRIPTION
**Pull request by TomBillingtonUK**
- Add StandardCarTestCATBSchema
- Mock Test Data
- Mock getUploadBatch
- Added new env vars
- Added ability to calculate the template id
- Moves some envvars to a new Config Provider

### CODE REVIEW

File: src/functions/sendCandidateResults/application/service/get-template-id.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+function isPass(activityCode: ActivityCode): boolean {
+  return activityCode === '1';
+}
+
+function isFail(activityCode: ActivityCode) : boolean {
+  return activityCode === '2' || activityCode === '3';
+}
+
+function throwError(activityCode: ActivityCode) : void {
```
Comment (szabikr) [inline src/functions/sendCandidateResults/application/service/get-template-id.ts]: isTerminated perhaps

**Review by szabikr — APPROVED** [review]

File: src/functions/sendCandidateResults/application/service/get-template-id.ts (hunk 1 above)
Comment (TomBillingtonUK) [reply src/functions/sendCandidateResults/application/service/get-template-id.ts]: Fixed

**Review by MarkPatt89 — APPROVED** [review]
Looks good, getNextUploadBatch in the real world has an `interface` property. Not need in your instance here though for mocking.

**Comment by TomBillingtonUK** [comment]
@MarkPatt89  - Will add that in in the next PR

**Pull request merged**